integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDynamicUpdatePart2.java

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2021, 2022, Oracle and/or its affiliates.
+// Copyright (c) 2021, 2022, 2023, Oracle and/or its affiliates.
```

Comment:
This is an incorrect copyright ;)...

The two years are the year that the file was created and the year that the file was most recently updated. So, this would be "2021, 2023,"

Reply:
Done. Thanks Ryan!